### DISCUSSION TITLE
Add (roof)fs scan support for OSes based on Yocto Project Framework

### DISCUSSION THREAD

**Discussion by ghost** [aquasecurity/trivy#5592] [Ideas]
### Description

Hello!

We have some devices that run Linux images based on the Yocto Project Framework. I tried to use Trivy to scan the file system structures, but it could not detect anything. Later on, I've checked the supported OS list in documents and it's not supported.

The devices we have the following architectures:
- Texas Instrument  Sitarra AM3352 Cortex A8
- NXP i.MX6DL Cortex A9
- Intel Apollo Lake Atom E3930

Would be possible to support scanning  Yocto Project Framework images in the future?

### Target

Filesystem

### Scanner

Vulnerability

**Comment by klaus-halfmann** [comment]
(+1) from my Side, having trivy in yocto (and/or some reasonable howto)
would help e.g. for https://rdkcentral.com/

**Comment by lyager** [comment]
+1 - I guess the Yocto FS is too minimal for Trivy to gather proper package information, right?
